verifier/credentialsConfig.go

```diff
@@ -53,39 +51,71 @@ func InitServiceBackedCredentialsConfig(repoConfig *config.ConfigRepo) (credenti
 			logging.Log().Warnf("Was not able to instantiate the config client.")
 		}
 	}
-	var serviceCache Cache = cache.New(CACHE_EXPIRY*time.Second, 2*CACHE_EXPIRY*time.Second)
 
-	scb := ServiceBackedCredentialsConfig{configClient: &configClient, serviceCache: serviceCache, initialConfig: repoConfig}
+	scb := ServiceBackedCredentialsConfig{configClient: &configClient, initialConfig: repoConfig}
+
+	err = scb.fillStaticValues()
+	if err != nil {
+		return nil, err
+	}
 
-	scb.fillStaticValues()
 	if repoConfig.ConfigEndpoint != "" {
-		chrono.NewDefaultTaskScheduler().ScheduleAtFixedRate(scb.fillCache, time.Duration(30)*time.Second)
+		_, err := chrono.NewDefaultTaskScheduler().ScheduleAtFixedRate(scb.fillCache, time.Duration(30)*time.Second)
```

Comment:
Should we link the refresh time to the cache expiration in some way?

Reply:
I didn't change this piece of code, but, I think, it should be fine, since the refresh will occur every 30 seconds, and the cache expiration time is 60 seconds. Or what do you think could be improved here?